### ISSUE TITLE
Quick TX fails on Raspberry Pi

### ISSUE THREAD

**Issue by fklassen** [appneta/tcpreplay#244]
From mail list:

```
I configured, compiled and installed it as pointed (./configure --enable-quick-tx) in INSTALL file on Raspberry Pi 2 with Raspbian OS.
It says "Illegal instruction" as a first output string when trying to use --quick-tx or qtx:mon0 options.
Works ok w/o the option.
```

**Comment by PavelBirulin** [comment]
Do you know if netmap supports wifi frames ?

**Comment by fklassen** [comment]
netmap questions belong here https://github.com/luigirizzo/netmap/issues

**Comment by fklassen** [comment]
Dropping support for Quick TX until a maintainer becomes available #357
